Sources/StreamChat/Database/DTOs/AttachmentDTO.swift

```diff
@@ -9,9 +9,9 @@ import Foundation
 class AttachmentDTO: NSManagedObject {
     /// An attachment id.
     @NSManaged private var id: String
-    var attachmentID: AttachmentId {
-        get { AttachmentId(rawValue: id)! }
-        set { id = newValue.rawValue }
+    var attachmentID: AttachmentId? {
```

Comment:
Why do we change this?

Reply:
This was dangerous already, and in the past I did get a crash on this sometimes. With the current setup now I was able to reproduce it, so I fixed it. Most likely it is because we can have a corrupted attachment when logging out or someting. So to be safe, it is better to avoid the force unwrap